app/models/tenure.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Tenure
+  include ActiveModel::Model
+
+  attr_accessor :typeCode, :typeDescription
+
+  class << self
+    def find_tenure_type(reference)
+      response = PlatformApis::TenancyInformation::Client.get_tenancy_information_by_property_reference(reference)
+
+      build(response["tenancies"].first["tenureType"])
+    end
+
+    def build(attribute)
```

Comment:
Please could we add a small comment with an example of the tenureType value, to see why we're splitting things.